modules/xresources.nix

```diff
@@ -26,9 +31,14 @@ in
       example = {
         "XTerm*faceName" = "dejavu sans mono";
         "Emacs*toolBar" = 0;
+        "rofi.color-active" = [ "#1d1f21" "#b5bd68" "#282a2e" "#969896" "#c5c8c6" ];
```

Comment:
Since this is a [deprecated](https://github.com/DaveDavenport/rofi/issues/756#issuecomment-357697381) way of configuring rofi colors I'd suggest using a different example. Perhaps

```nix
"XTerm*charClass" = [ "37:48" "45-47:48" "58:48" "64:48" "126:48" ];
```

?